_code/InSituDiff/R/NeighborhoodCalculations.R

```diff
@@ -27,6 +27,10 @@ getNeighborhoodExpression <- function(x, neighbors, makedense = TRUE) {
 #' @return sparse matrix holding neighbor assignments
 #' @export
 getNeighbors <- function(xy, neighbors, tissue, k, radius, verbose) {
```

Comment:
May be useful to provide default values of NULL for all of these since they're not all required, for ease of using this function directly.